clients/rospy/src/rospy/core.py

```diff
@@ -343,12 +343,13 @@ def configure_logging(node_name, level=logging.INFO):
         logfilename_remap = mappings['__log']
         filename = os.path.abspath(logfilename_remap)
     else:
-        # fix filesystem-unsafe chars
-        filename = node_name.replace('/', '_') + '.log'
-        if filename[0] == '_':
-            filename = filename[1:]
-        if not filename:
-            raise rospy.exceptions.ROSException('invalid configure_logging parameter: %s'%node_name)
+        # fix filesystem-unsafe chars
+        suffix = '.log'
+        filename = node_name.replace('/', '_') + suffix
+        if filename[0] == '_':
+            filename = filename[1:]
+        if filename == suffix:
+            raise rospy.exceptions.ROSException('invalid configure_logging parameter: %s'%node_name)
```

Comment:
ah sorry this happens because of my editor switching to unix endlines by default...
I ll put back the windows endlines there to get a proper diff.